### PR TITLE
AT_036.1_Java | Entity > Calendar: Edit and save a draft record

### PR DESCRIPTION
**Pull request by AnastasiiaPV**
test Edit Save Draft [Record]
(https://trello.com/c/rvZUbSMS/97-us036-entity-calendar-edit-and-save-a-draft-record)

### CODE REVIEW

File: src/test/java/EntityCalendar2Test.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import base.BaseTest;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import utils.ProjectUtils;
+import utils.TestUtils;
+
```
Comment (Jenny-ai) [inline src/test/java/EntityCalendar2Test.java:9]: make 
import static utils.ProjectUtils.*;
import static utils.TestUtils.*;
and remove ProjectUtils. and TestUtils through out the test.

File: src/test/java/EntityCalendar2Test.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        TestUtils.scrollClick(getDriver(), findElement(By.xpath("//p[contains (text(), 'Calendar')]")));
+        getDriver().findElement(By.xpath("//*[@class='card-icon']")).click();
+
+        getDriver().findElement(By.id("string")).sendKeys(str);
+        getDriver().findElement(By.id("text")).sendKeys(text);
+        getDriver().findElement(By.id("date")).click();
+        getDriver().findElement(By.id("datetime")).click();
+        getDriver().findElement(By.id("int")).sendKeys(in);
+        getDriver().findElement(By.id("decimal")).sendKeys(decimal);
+
```
Comment (Jenny-ai) [inline src/test/java/EntityCalendar2Test.java:24]: you can remove getDriver() before findElements(), because it is extracted in BaseTest

Comment (AnastasiiaPV) [reply src/test/java/EntityCalendar2Test.java:24]: Thank you. getDriver() has been deleted.

File: src/test/java/EntityCalendar2Test.java (hunk 1 above)
Comment (AnastasiiaPV) [reply src/test/java/EntityCalendar2Test.java:9]: My IntelliJ does not allow me to do this. 
It changes from import static utils.ProjectUtils back to import utils.ProjectUtils;

Comment (Jenny-ai) [reply src/test/java/EntityCalendar2Test.java:9]: I imported * from those two classes. It works in this case

**Review by Jenny-ai — APPROVED** [review]

File: src/test/java/EntityCalendar2Test.java (hunk 1 above)
Comment (AnastasiiaPV) [reply src/test/java/EntityCalendar2Test.java:9]: I did it as well but it was canceled by IntelliJ

Comment (Jenny-ai) [reply src/test/java/EntityCalendar2Test.java:9]: ok

**Review by ArinaJur — APPROVED** [review]
LGFM

**Pull request merged**